### ISSUE TITLE
Insults are not refreshed (always/often the same insult is shown)

### ISSUE THREAD

**Issue by EvilInsultGenerator** [EvilInsultGenerator/website#23]
When asking the API for a random insults, it often delivers the same insults over and over. 
Some users (website & API) and myself noticed this.

**Comment by WolvTMG** [comment]
Yep, this is happening for me as well.

**Comment by EvilInsultGenerator** [comment]
Thankfully I got a hint from a user and implemented it in https://github.com/EvilInsultGenerator/website/commit/6d0f28b307e8b9d95d68202c5aadc78bee4b42e2. 🙏

**Comment by INFINITE31** [comment]
Still the same issue wonder

**Comment by EvilInsultGenerator** [comment]
> Still the same issue

Maybe caching might be an issue. 🤔

**Comment by INFINITE31** [comment]
Is it working for you guys? because its not working for me though

**Comment by EvilInsultGenerator** [comment]
> Is it working for you guys? because its not working for me though

It's not working/refreshing for me too. Maybe there is some server-side caching of which I am unaware. 🤔
If anyone has a suggestion, please let me know.

**Comment by EvilInsultGenerator** [comment]
One user sent this:

> I found a workaround by adding a random get value and string at the end of the api call.  Literally adding "&something=" with a random 20 character string.  So the problem lies in the api requests getting cached, I'm guessing server side because the python API calls don't have a cache.

**Comment by Natemo6348** [comment]
I've been having this issue as well, though I've only noticed this from the API side of things

**Comment by EvilInsultGenerator** [comment]
> I've been having this issue as well, though I've only noticed this from the API side of things

Do you have an idea what might cause the issue? Any help is highly appreciated.

**Comment by Natemo6348** [comment]
> > I've been having this issue as well, though I've only noticed this from the API side of things
> 
> Do you have an idea what might cause the issue? Any help is highly appreciated.

I don't, but I can definitely look at it and see if I can figure it out

**Comment by Natemo6348** [comment]
It seems like the issue has been resolved, at least from the API, insults are refreshing for me now

**Comment by EvilInsultGenerator** [comment]
Yes, it looking fine now. Maybe it was a server issue after all. 🤔
